### PR TITLE
Add site update news: the move to Statocles and further clean-up

### PR DESCRIPTION
**Pull request by ehuelsmann**
Note that we removed most JS and migrated the menu and other parts
to CSS-only solutions.

### CODE REVIEW

File: blog/2019/05/19/new-site.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: New site engine for rexify.org
+date: 2019-05-19 00:00:00
+tags: news
+---
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: I guess we could use `author` in frontmatter too.

File: blog/2019/05/19/new-site.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+[Statocles](http://preaction.me/statocles/).
+
+---
+
+The switch achieves multiple goals, all of which are greatly welcomed:
+
+* Simplified maintenance
+* Faster page loads
+* Hosted on GitHub Pages
+* Maintenance distributed over more people
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: This feels like related to the first point, so they may be merged.

File: blog/2019/05/19/new-site.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: That looks like an accidental double new line.

File: blog/2019/05/19/new-site.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+As the initial product, we've migrated all pages from the old site,
+as much as possible, with the following notable differences:
+
+* All pages were converted to Markdown where possible
+* The API documentation was not migrated (instead, we defer
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: In the current form the sentence doesn't make much sense:

"The API documentation was _not_ migrated [...] to MetaCPAN"

Perhaps this can be rewritten as:

```
The [API documentation](https://metacpan.org/release/Rex) simply links to [MetaCPAN](https://metacpan.org/)
```

File: blog/2019/05/19/new-site.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  [browsing our API docs](https://metacpan.org/release/Rex) to
+  [MetaCPAN](https://metacpan.org/))
+* The news pages were not migrated and re-adding news to our site
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: `s/pages/entries/`

File: blog/2019/05/19/new-site.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  is left as a post-migration step to be handled over the
+  coming weeks
+
+The contribution instructions in the GitHub repository have been
```
Comment (ferki) [inline blog/2019/05/19/new-site.md]: This might be setting too high expectations; we just updated the README, but there's no contributoin guide yet :)

File: blog/2019/06/09/site-update.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+Over the course of the past weeks many clean-ups have been
+done, reaping the benefits provided by our new
+[Statocles](https://preaction.me/statocles)-based site:
+
+---
+
+* Added back the ability maintain News on the site
+* Remove "Fork me on GitHub" banner for improved mobile usability
+* Cleaned up GitHub repository ('master' branch) from all unused parts
```
Comment (ferki) [inline blog/2019/06/09/site-update.md]: I'm not sure if the removal is complete in the sense, in the sense of

> all unused parts

File: blog/2019/06/09/site-update.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+* Removed almost all JavaScript components, due to
```
Comment (ferki) [inline blog/2019/06/09/site-update.md]: Perhaps we can slightly rephrase this as "Removed all JavaScript components we could".

File: blog/2019/06/09/site-update.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+  * Moved the menu from JavaScript to CSS+HTML
+  * Moved the 'Get Rex' page to CSS+HTML (pre-migration)
+  * browserJS not used in recent versions of the site
+  * jQuery used to be used by now-removed administrative part of the site
```
Comment (ferki) [inline blog/2019/06/09/site-update.md]: `s/site/old site/`? To make it more clear.

File: blog/2019/06/09/site-update.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+  * bootstrap didn't make any difference on simulated mobile devices
```
Comment (ferki) [inline blog/2019/06/09/site-update.md]: - `s/bootstrap/unused bootstrap plugins/` to make clear we didn't remove bootstrap (the whole library), just its javascript plugins (which were unused)
- I believe jquery was also a dependency for bootstrap plugins, so perhaps the last two lines can be switched in order

File: blog/2019/05/19/new-site.md (hunk 1 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: Added.

File: blog/2019/05/19/new-site.md (hunk 2 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: Ok. Merged the two in follow-up commit.

File: blog/2019/05/19/new-site.md (hunk 3 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: Removed.

File: blog/2019/05/19/new-site.md (hunk 4 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: Yup. Rephrased.

File: blog/2019/05/19/new-site.md (hunk 6 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: The difference between the sales and the programmer :-)

File: blog/2019/06/09/site-update.md (hunk 7 above)
Comment (ehuelsmann) [reply blog/2019/06/09/site-update.md]: Rephrased to remove the inclination to add the "all" trigger-word.

File: blog/2019/06/09/site-update.md (hunk 9 above)
Comment (ehuelsmann) [reply blog/2019/06/09/site-update.md]: No problem with it; "used to be used" seems to imply it; now it's explicit.

File: blog/2019/06/09/site-update.md (hunk 10 above)
Comment (ehuelsmann) [reply blog/2019/06/09/site-update.md]: Hmm. But there's no *bootstrap* file being loaded anymore. which to me says "all bootstrap JS" (as in: not *just* the plugins). Are we seeing different things?

File: blog/2019/05/19/new-site.md (hunk 5 above)
Comment (ehuelsmann) [reply blog/2019/05/19/new-site.md]: Done.

File: blog/2019/06/09/site-update.md (hunk 8 above)
Comment (ehuelsmann) [reply blog/2019/06/09/site-update.md]: Changed to proposed wording.
